tools/wptrunner/wptrunner/executors/base.py

```diff
@@ -243,7 +243,7 @@ def logger(self):
         return self.executor.logger
 
     def get_hash(self, test, viewport_size, dpi):
-        timeout = test.timeout * self.timeout_multiplier
+        test.timeout * self.timeout_multiplier
```

Comment:
This is dead code, just remove it.

Reply:
Good catch. I swear this was a function call last night.